guides/process/scheduling/how_to_timesheet.md

```diff
@@ -51,24 +51,49 @@ Some categories should be used where there is significant time spent, i.e. 2 hou
 
 ## How to submit your timesheet
 
-Timesheets are submitted in Harvest, you will be given access on your first day. 
+Timesheets are submitted in Harvest, you will be given access on your first day.
 
-In the time tab at the top of the page toggle to the week section. 
+In the time tab at the top of the page toggle to the week section.
 
 If it’s your first time sheet, you will need to scroll down and select add row:
 
 
 A pop up window comes up for you to select from the drop down menu the client project you are currently working on including any Made Tech internal projects.
 
-To add additional projects, you need to select the + New Row button at the bottom indicated below. 
+To add additional projects, you need to select the + New Row button at the bottom indicated below.
 
 
 
 Once you have selected the projects they will be listed. In the week view you will need to input the number of hours spent on each client project including any Made Tech ones. Total time needs to add up to 35 hours for full time team members or agreed prorated hours.
 
-For new team members joining mid-week, the total hours for the first week will be what has been worked since you started i.e. 21 hours for 3 days. 
+For new team members joining mid-week, the total hours for the first week will be what has been worked since you started i.e. 21 hours for 3 days.
+
+When you enter time, press the green save button at the bottom to save your timesheet.
+
+
+## Timesheet approval process
+
 
-When you enter time, press the green save button at the bottom to save your timesheet. 
+Once a team member has submitted their timesheet on Friday by 12pm, Delivery Managers and Service Area owners need to approve their teams timesheets by 1pm.
+
+Why we need to approve timesheets
+Timesheet data directly drives our client invoicing, so it’s important it’s accurate and complete at the end of every week. It will also save you time when checking invoice accuracy at the end of the month. When approving timesheets for the week we need to:
+* Check your team has completed their timesheet in full; and has accurately recorded the time they spent on the project (inc. absences like illness and holidays)
+* Make sure their project codes are correct (inc. for unbilled time)
+* Account for any differences between the team’s Forecast time and their actual time spent and report this to the Operations team
+
+The review and approval process should take 5-10 minutes; and picking up any errors or unexplained differences saves the business a lot of time!
+Here are the instructions to approve someone's time:
+* In [Harvest](https://madetech.harvestapp.com/time/week) at the top select Time from the heading options
+* From the subheadings select Pending Approval
```

Comment:
Try putting these "actions" in codeblock backticks and they are easier to follow `Pending Approval` `